packages/xo-server/src/xo-mixins/vmware/index.mjs

```diff
@@ -139,7 +139,14 @@ export default class MigrateVm {
       }
       coldChainsByNodes[key] = chainCopy
     })
-
+    // ensure the session stays alive
```

Comment:
Instead of doing a manual "ping", is there no way to pass a "keep-alive" option?

Reply:
Related question: is `esxi.getTransferableVmMetadata()` the smallest/fastest call we can do for the keep-alive?

Reply:
@MathieuRA  I didn't find this in the doc. 
@julien-f  no i do a single fetch  instead of making a lot of queries for the VM ( <hich load all the VM info),